bot/constants.py

```diff
@@ -25,7 +24,7 @@
 except ModuleNotFoundError:
     pass
 
-log = logging.getLogger(__name__)
+log = logging.getLogger(__name__)  # use logging.getLogger (not bot.utils.logging.get_logger) to prevent circular import
```

Comment:
I'm not sure this is the best option here, there was some discussion on discord on this starting here https://discord.com/channels/267624335836053506/635950537262759947/894235145362616390, and it seems the conclusion was that the logs here don't work anyway because this is run before logging is set up.

Reply:
This has been resolved with the little re-write :+1: